File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { BluefinShieldconexMgmtSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinShieldconexMgmtSDK.test()
    equal(null !== testsdk, true)
  })

})

File: dart/test/exists_test.dart
import 'harness.dart';

import '../lib/BluefinShieldconexMgmtSDK.dart';

void tests() {
  describe('exists', () {
    test('test-mode', (t) async {
      final testsdk = BluefinShieldconexMgmtSDK.test();
      equal(true, null != testsdk);
    });
  });
}
